tests/functional/timer/test_job_operations.py

```diff
@@ -5,15 +5,15 @@
 import pytest
 from globus_sdk._testing import RegisteredResponse, load_response, load_response_set
 
-from globus_cli.commands.timer._common import JOB_FORMAT_FIELDS
+from globus_cli.commands.timer._common import TIMER_FORMAT_FIELDS
 
-# NOTE: this is not quite the same as the "normal" job responses from
+# NOTE: this is not quite the same as the "normal" timer responses from
```

Comment:
Nit: I might not change this note until we've updated the APIs, since technically it _is_ a job response today (and hopefully once we make it a proper timer response, we can return and remove the note entirely!)

Reply:
Is this something you want changed?

Reply:
I didn't request a change b/c I didn't feel strongly enough (or that the impact was significant enough) to insist on it, only that it registered on my radar. I leave it to your judgment, but will not be put out if you leave it as is.